schema-engine/sql-schema-describer/src/lib.rs

```diff
@@ -34,25 +34,13 @@ use std::fmt::{self, Debug};
 /// A database description connector.
 #[async_trait::async_trait]
 pub trait SqlSchemaDescriberBackend: Send + Sync {
-    /// List the database's schemas.
-    async fn list_databases(&self) -> DescriberResult<Vec<String>>;
-
-    /// Get the databases metadata.
-    async fn get_metadata(&self, schema: &str) -> DescriberResult<SqlMetadata>;
```

Comment:
ℹ️ This method was marked somewhere already as "do we really need that". And indeed it was unused and I deleted a whole bunch of related dead code.